frontend/src/components/CompareModels.vue

```diff
@@ -36,6 +36,20 @@
       </div>
       <br />
       <br />
+      <h3 class="title is-size-3">Historical comparisions</h3>
+      <div>
+        <b>Human-GEM</b> was created by unifying several model lineages, including
+        <b
+          ><a href="https://europepmc.org/article/PMC/2013923"
+            >Human Metabolic Reaction series (HMR)</a
+          ></b
+        >
+        and <b><a href="https://vmh.life/#home">Recon</a></b
+        >. Since then, Human-GEM has been extensively curated and new versions are continuously
+        published. The tables below give an overview of how previous versions of Human-GEM relate to
```

Comment:
```suggestion
        published. The tables below give an overview of how Human-GEM 1.2.0 relate to
```

Reply:
I think this makes sense, but I wonder if for example
```txt
The tables below give an overview of how version 1.0.2 of Human-GEM relates...
```
would make it even clearer that it's not the current version?

Reply:
Yes, I prefer your suggestion!

Reply:
Fixed in c1e68ab